frontend/src/bundles/templates-page/template-api.ts

```diff
@@ -91,6 +103,26 @@ class TemplateApi extends HttpApi {
 
         return await response.json<TemplateGetAllResponseDto>();
     }
+
+    //TODO this is stub since there is no time to handle this problem properly please ignore it
```

Comment:
Why add it if it's not finished?

Reply:
because it's not working without this stub. maybe in future when we add proper query handling in http requests, I will remove it.